### PR TITLE
Cr 922 locust uac paging

### PR DESCRIPTION
**Pull request by PeterBochel**
Minor changes for Locust test instance support:

- Added 'InsecureSkipVerify' option for ignoring x509 certificate checking errors.
- Bug fix for small data sets, which were generating too much data

### CODE REVIEW

File: go/src/github.com/census/integration/sampleGenerator/sampleGenerator.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ import (
 	"sync"
 	"sync/atomic"
 	"time"
+    "crypto/tls"
```
Comment (rpcatling) [inline go/src/github.com/census/integration/sampleGenerator/sampleGenerator.go:33]: formatting!

File: go/src/github.com/census/integration/sampleGenerator/sampleGenerator.go [hunk 2]
```diff
@@ -138,6 +141,7 @@ func main() {
 		// Work out how many events we want this thread to generate
 		quantityForThread := config.Quantity / config.ThreadCount
 		if threadNumber == config.ThreadCount {
+		    // Let final thread generate enough to bring us to the requested total
```
Comment (rpcatling) [inline go/src/github.com/census/integration/sampleGenerator/sampleGenerator.go:144]: formatting  ... "go fmt"

File: go/src/github.com/census/integration/sampleGenerator/sampleGenerator.go (hunk 1 above)
Comment (PeterBochel) [reply go/src/github.com/census/integration/sampleGenerator/sampleGenerator.go]: Have ran formatting

File: go/src/github.com/census/integration/sampleGenerator/sampleGenerator.go (hunk 2 above)
Comment (PeterBochel) [reply go/src/github.com/census/integration/sampleGenerator/sampleGenerator.go]: as before, now formatted

**Pull request merged**